src/inmanta/plugins.py

```diff
@@ -399,8 +400,8 @@ def resolve_type(self, plugin: "Plugin", resolver: Namespace) -> inmanta_type.Ty
     def validate(self, value: object) -> bool:
         """
         Validate that the given value can be passed to this argument.  Returns True if the value is known
-        and valid, False if the value is unknown, and raises a ValueError is the value is not of the
-        expected type.
+        and valid, False if the value is unknown, and raises a :py:class:`inmanta.ast.RuntimeException`
+        if the value is not of the expected type.
```

Comment:
Good fix. I was surprised to see this, I'd have sworn `Type.validate` raised a `ValueError`, but looks like I was wrong.